en/micro-integrator/docs/use-cases/tutorials/storing-and-forwarding-messages.md

```diff
@@ -276,22 +256,11 @@ Let's test the use case by sending a simple client request that invokes the serv
     java -jar Hospital-Service-2.0.0-EI7.jar
     ```
 
-#### Start the Message Broker runtime
+#### Start the RabbitMQ Broker
     
-To start the Message Broker:
-
-1.  Open a terminal and navigate to the `EI_6.6.0_HOME/wso2/broker/bin` directory.
-2.  Execute the following command to run the message broker. 
-
-    ```bash tab='On MacOS/Linux/Centos'
-    sh wso2server.sh
-    ```
-
-    ```bash tab='On Windows'
-    wso2server.bat
-    ```
+Make sure that you have installed and started a RabbitMQ server instance for the Micro-Integrator to communicate with.
 
-    See the [WSO2 EI 6.6.0 documentation](https://docs.wso2.com/display/EI660/Running+the+Product) for more information on how to run the Message Broker profile.
+    See the [RabbitMQ documentation](https://www.rabbitmq.com/download.html) for more information on how to install and run the product.
```

Comment:
Users should be able to follow the 'Connecting to RabbitMQ' doc for this one, right?